README-dev.md

```diff
@@ -21,6 +21,7 @@ Before starting, ensure you have the following tools installed:
 After cloning the repository, you need to fetch the submodules:
 
 ```sh
+export GIT_LFS_SKIP_SMUDGE=1
 git submodule update --init --recursive
 ```
```

Comment:
nice. but it's probably just needed the first time you pull the submodules right? could you mention that?

Reply:
sorry, I missed that, will do.